docs/cli-reference/dfx-deps.md

```diff
@@ -0,0 +1,111 @@
+# dfx deps
+
+Use the `dfx deps` command with flags and subcommands to pull dependencies from mainnet and deploy locally.
```

Comment:
Are we always calling this "the mainnet" now? @jessiemongeon1 
```suggestion
Use the `dfx deps` command with flags and subcommands to pull dependencies from the mainnet and deploy locally.
```

Reply:
Yes, either 'the mainnet' or 'the IC/Internet Computer mainnet'.